### ISSUE TITLE
Problem with team partition

### ISSUE THREAD

**Issue by iamceph** [MC-Minigames/BedWars#10]
Hi. I have your BedWars plugin on my server. In some cases, we have in Blue team 5 players and in other teams only 1 player. Any way to fix it? (No player used the settings team).
Thank you.

**Comment by PantallazoAzul** [comment]
Yes, my problem is: If all players start in the same team, the game anyways begins 😮 no enemies! 😢
